deps/rabbit/src/rabbit_db.erl

```diff
@@ -88,7 +111,9 @@ force_reset_using_mnesia() ->
     ?LOG_DEBUG(
       "DB: resetting node forcefully",
       #{domain => ?RMQLOG_DOMAIN_DB}),
-    rabbit_mnesia:force_reset().
+    ok = rabbit_mnesia:force_reset(),
+    ok = rabbit_node_monitor:reset_cluster_status(),
```

Comment:
I think `rabbit_node_monitor:reset_cluster_status()` should be called from `force_reset()` just after Mnesia ops:
```
force_reset() ->
 force_reset_using_mnesia(),
 ok = rabbit_node_monitor:reset_cluster_status().
```

Reply:
Good catch! Thank you!